### ISSUE TITLE
Indentation broken

### ISSUE THREAD

**Issue by FunTimeCoding** [FriendsOfPHP/PHP-CS-Fixer#225]
Like the example shows, Ive tried both with 2 spaces and with 0 spaces indented, using the latest php-cs-fixer version it wont fix the indentation. what version does it work with still?

shiin  ~  .dotfiles  code  php  cat Test.php
<?php
namespace TEST;
class Test
{
  public function test()
  {
    echo "test";
  }
}
shiin  ~  .dotfiles  code  php  php-cs-fixer self-update  
php-cs-fixer updated.
shiin  ~  .dotfiles  code  php  php-cs-fixer fix Test.php --level=psr2 --verbose --dry-run --diff
shiin  ~  .dotfiles  code  php  cat Test.php
<?php
namespace TEST;
class Test
{
  public function test()
  {
    echo "test";
  }
}
shiin  ~  .dotfiles  code  php  vim Test.php
shiin  ~  .dotfiles  code  php  cat Test.php
<?php
namespace TEST;
class Test
{
public function test()
{
echo "test";
}
}
shiin  ~  .dotfiles  code  php  php-cs-fixer fix Test.php --level=psr2 --verbose --dry-run --diff

**Comment by fghber** [comment]
I have the same problem. Is there a suggested way to deal with this problem, e.g. convert all tabs to spaces?

**Comment by keradus** [comment]
please elaborate what you wish to achieve in first place?

**Comment by fghber** [comment]
I would like to have fixed the indentation of some PHP files with mixed tabs and spaces (and maybe even mixed line breaks) with php-cs-fixer 2. NetBeans can format the source nicely, but it is tedious to open each file manually to apply the source formatting. 
I have tried something like this

```
return PhpCsFixer\Config::create()
    ->setRules([
        '@PSR2' => true,
        'indentation_type' => true,
    ])
    ->setIndent("\t")
    ->setLineEnding("\n")
```

I've also tried adding -indentation in the "Fixers" field (but maybe this is the very problem).
